optuna/visualization/matplotlib/_utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from optuna.distributions import LogUniformDistribution
+from optuna.trial import FrozenTrial
+from optuna.visualization.matplotlib import _matplotlib_imports
+
+__all__ = ["is_available"]
+
+
+def is_available() -> bool:
+    """Returns whether visualization with `matplotlib` is available or not.
```

Comment:
A nit, but a single backtick corresponds to italic. Please use double backticks for consistency. 
```suggestion
    """Returns whether visualization with ``matplotlib`` is available or not.
```